scripts/get-dep-lib.sh

```diff
@@ -169,8 +169,9 @@ for package in "${PACKAGES[@]}" ; do
 			git_checkout_library breeze-icons $CURRENT_BREEZE_ICONS https://github.com/kde/breeze-icons.git
 			;;
 		googlemaps)
-			git_checkout_library googlemaps master https://github.com/Subsurface/googlemaps.git
-			;;
+			#git_checkout_library googlemaps master https://github.com/Subsurface/googlemaps.git
+			git_checkout_library googlemaps master https://github.com/vladest/googlemaps.git
+      ;;
```

Comment:
yeah - he had not maintained the code for quite a while, so I added our own initial Qt6 port (plus a couple other fixes, IIRC) to our fork. Glad he's back working on the code.
When I looked his stuff, though, it failed to compile on Qt 6.7.2 - that has been a few weeks, maybe he addressed those issues? I'm 99% sure I had to make another couple of fixes...